README.md

```diff
@@ -841,24 +841,15 @@ index 49a2661..1804a01 100644
              - "traefik.http.routers.${PROJECT_NAME}-frontend-unsecure.middlewares=redirect-to-https@file"
 
 -    postgres:
--        build: services/postgres
+-        image: postgres:16
 -        environment:
 -            - POSTGRES_USER=app
 -            - POSTGRES_PASSWORD=app
 +    mysql:
-+        build: services/mysql
++        image: mysql:8
          volumes:
 -            - postgres-data:/var/lib/postgresql/data
 +            - mysql-data:/var/lib/mysql
```

Comment:
do we need to add the `MYSQL_ALLOW_EMPTY_PASSWORD` env var from the previous dockerfile?

Reply:
I guess 👍🏼